week3/ml_decision_tree_sample/generator_model_card.py

```diff
@@ -0,0 +1,83 @@
+import hydra
+from omegaconf import DictConfig
+import os, sys
+import model_card_toolkit as mctlib
+import uuid
+from datetime import date
+from predictor import create_confusion_matrix, create_graph_importance, create_graph_svc
+from prepare_data import prepare_data
+
+
+def generate_model_card(max_depth, min_samples_leaf, min_samples_split, n_estimators,
+                        random_state, patch_to_file,name, model_card_overview,
+                        owner_name,owner_contact, reference1, reference2,
+                        risk_name,risk_mitigation_strategy, limitation,consider_use_cases,
+                        consider_user1,consider_user2, consider_user3,model_card_html_name):
+    X_test = prepare_data(patch_to_file)[1]['x']
+    model_card_output_path = f'{os.path}'.replace(os.path.basename(sys.argv[0]), '')
```

Comment:
please never do this, pass model_card_output_path as function input

Reply:
What should use instead of this?

Reply:
I wrote here, use some type of input, via typer or argparse

Reply:
this code is would be hard to test and modify

Reply:
chenged